cpp/src/toppra/constraint/linear_joint_velocity.hpp

```diff
@@ -6,7 +6,24 @@
 namespace toppra {
 namespace constraint {
 
-/// A Joint Velocity Constraint class.
+/**
+\brief A Joint Velocity Constraint class.
+
+This class implement the case of constant velocity limits but can be derived to
+achieve varying velocity limits. E.g.
+\code
+class LinearJointVelocityVarying : public LinearJointVelocity {
+public:
+  LinearJointVelocityVarying(...) : LinearJointVelocity (nDof) {}
+protected:
+  void computeVelocityLimits(value_type time)
+  {
+    m_lower = ...;
+    m_upper = ...;
+  }
+};
+\endcode
```

Comment:
@thummel42 here is how to use this.

Reply:
Awesome thanks!! Looking forward to the merge.